spec/requests/api/v1/participant_declarations_spec.rb

```diff
@@ -125,4 +125,95 @@ def build_params(attributes)
       end
     end
   end
+
+  describe "get" do
+    let(:token) { LeadProviderApiToken.create_with_random_token!(cpd_lead_provider: cpd_lead_provider) }
+    let(:bearer_token) { "Bearer #{token}" }
+
+    let(:participant_declaration) do
+      create(:participant_declaration,
+             user: ect_profile.user,
+             cpd_lead_provider: cpd_lead_provider,
+             course_identifier: "ecf-induction")
+    end
+
+    let!(:profile_declaration) do
+      create(:profile_declaration,
+             participant_declaration: participant_declaration,
+             participant_profile: ect_profile)
+    end
+
+    context "when authorized" do
+      context "when there is a non eligible declaration" do
+        let(:expected_response) do
+          JSON.parse(<<-DATA)
+            {"data":
+              [
+                {
+                  "id":"#{participant_declaration.id}",
+                  "type":"participant_declaration",
+                  "attributes": {
+                    "participant_id": "#{ect_profile.user.id}",
+                    "declaration_type": "started",
+                    "declaration_date": "#{participant_declaration.declaration_date.rfc3339(3)}",
+                    "course_identifier": "ecf-induction",
+                    "eligible_for_payment": false
+                  }
+              }]
+            }
+          DATA
```

Comment:
JSON.parse takes string and returns a ruby hash.

Why don't we skip that, and `<<-Data` steps and define our hash in pure ruby?